modules/ROOT/pages/install-helm.adoc

```diff
@@ -152,11 +152,11 @@ kubectl apply -f <filename>.yaml
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: <name>
+  name: rtf-agent
   namespace: <additional-namespace>
 subjects:
   - kind: ServiceAccount
-    name: rtf-namespace
+    name: rtf-agent
     namespace: rtf-namespace
```

Comment:
the `namespace` value we can change it to `rtf` as we have that restriction for now.